cid/helpers/quicksight/__init__.py

```diff
@@ -914,7 +919,7 @@ def get_dataset_last_ingestion(self, dataset_id) -> str:
         except self.client.exceptions.AccessDeniedException:
             return '<YELLOW>AccessDenied<END>'
         if not ingestions:
-            return None
+            return None #todo fix this using dataset import type. None could be if refresh never ran too, no ? 
```

Comment:
this can only be if there no ingestions yet. if it is DIRECT, a DATA_SET_NOT_SPICE error will be in the  results.

Reply:
Got it, Thank you! removed the comment.